src/sklearn_raster/estimator.py

```diff
@@ -618,9 +627,16 @@ def inverse_transform(
             allow_cast=allow_cast,
             check_output_for_nodata=check_output_for_nodata,
             nan_fill=0.0,
+            set_attrs={
+                "source_method": self._method_name(wrapped_func),
+            },
```

Comment:
Super minor, but if you remove the comma at the end, I think this fits on one line (like all the rest)